settings.gradle.kts

```diff
@@ -14,6 +14,7 @@ pluginManagement {
         maven("https://repo.nea.moe/releases")
         maven("https://repo.sk1er.club/repository/maven-releases/")
         maven("https://maven.deftu.dev/releases")
+        maven("https://repo.polyfrost.org/releases")
```

Comment:
is there no other repo to get this from?

Reply:
wdym? this is the repo in which has a fork that supports the java version as bloosom changed how it works on newer java versions in 2.x

Reply:
This is the changes that Polyfrost made https://github.com/KyoriPowered/blossom/compare/master...Polyfrost:blossom:master

Reply:
if the issue is about uptime of polyfrost infrastructure, I can put a mirror of it on my own maven (which is already used for the RPC)